packages/vite/src/vite-node.ts

```diff
@@ -143,7 +143,7 @@ export async function initViteNodeServer (ctx: ViteBuildContext) {
   // Serialize and pass vite-node runtime options
   const viteNodeServerOptions = {
     baseURL: `${protocol}://${host}:${port}/__nuxt_vite_node__`,
-    rootDir: ctx.nuxt.options.rootDir,
+    root: ctx.nuxt.options.srcDir,
```

Comment:
I guess would also need to set the root for the Vite server to ensure the path resolution's consistency

https://github.com/nuxt/framework/blob/b0bf25cf51e390ffbc15bdac1a895b8a433f5dd8/packages/vite/src/vite.ts#L34

Currently we seems to use the implicit `process.cwd()`

Reply:
```suggestion
    root: ctx.ssrServer.config.base.root,
```

After that we might reuse the root option from the vite server dierectly

Reply:
Interesting. I think we are setting it here, right?

https://github.com/nuxt/framework/blob/e55938fe6ec8e71800a1c8bc6842a379183556f0/packages/schema/src/config/vite.ts#L14-L17

Reply:
Oh yeah, then it totally makes sense to use `srcDir` 👍

Reply:
They are safe references. I think it is better to have single source of trust `srcDir` referencing everywhere.

Reply:
Okay! 👍